app/src/main/java/aloksharma/ufl/edu/stash/ServerAccess.java

```diff
@@ -106,10 +124,104 @@ private Double getBankBalance(String username, String password, BankName
         postArgs.add(new BasicNameValuePair("username", username));
         postArgs.add(new BasicNameValuePair("password", password));
         postArgs.add(new BasicNameValuePair("type", bankName.toString()));
-
         return plaidPostRequest(plaidUrl, postArgs);
     }
 
+   /******************Add Stash Functionality******************************************/
+    public void addStash(String StashName, String StashTargetDate, String StashGoal){
+        //Stub to create Stash
+        Log.d("CreateStashLog1",StashName);
+        Log.d("CreateStashLog2",StashTargetDate);
+        Log.d("CreateStashLog3",StashGoal);
+
+        //Send to Parse Database
+        final ParseObject Stash = new ParseObject("Stash");
+
+        Stash.put("StashName", StashName);
+        Stash.put("StashTargetDate", StashTargetDate);
+        Stash.put("StashGoal", StashGoal);
+
+
+        //Link the ParseUser object with the Stash object
+        ParseUser currentUser = ParseUser.getCurrentUser();
+
+        Stash.put("user", ParseUser.getCurrentUser());     //create a user relation with the current user
+
+        //Stash.saveInBackground();
+
+        Stash.saveInBackground(new SaveCallback() {
+
+            @Override
+            public void done(ParseException e) {
+                if (e == null) {
+                    Log.i("Success","11");
+
+                } else {
+                    Log.i("Fail", "22");
+                }
+            }
+
+        });
+        /******************End of Add Stash Functionality******************************************/
+
+
+        /******************My Stashes Functionality******************************************/
+       //Populating the list of Stash Fields; that query should be via a button
+
+        // Create query for objects of type "Post"
+        ParseQuery<ParseObject> query = ParseQuery.getQuery("Stash");
+
+        // Restrict to cases where the user is the current user.
+        query.whereEqualTo("user", ParseUser.getCurrentUser());
+
+
```

Comment:
You have quite some stray empty lines in between your code. Try to have your code in logical blocks, and keep stray empty lines only when needed for readability. Its just good coding practice.